### PR TITLE
Email spoofing best practices exchange

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-76b61de814/how-to/)

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- **Time needed:** Approximately 15 minutes
+- **Tools required:**  Email access
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+If you would prefer a video tutorial, see [Exchange- Spoofing: How to identify & protect your organization](https://emailhelp.rackspace.com/l/how-to-prevent-email-spoofing).
+
+### What is email spoofing?
+
+The word **spoof** means **falsified**. A spoofed email is one in which the sender purposely alters parts of the email to make the message appear as though it was authored by someone else. Commonly, the sender’s name and email address and the body of the message are formatted to appear from a legitimate source. Sometimes, the spoofer makes the email appear to come from a private citizen.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: Sometimes the spoofer makes the email appear to come from a private citizen.

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+### Prerequisites
+
+- **Applies to:** Administrator and User
+- **Difficulty:** Moderate
+- **Time needed:** Approximately 15 minutes
+- **Tools required:**  Email access
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+If you would prefer a video tutorial, see [Exchange- Spoofing: How to identify & protect your organization](https://emailhelp.rackspace.com/l/how-to-prevent-email-spoofing).
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: Exchange - Spoofing (add space on left side of hyphen (to match what is shown on YouTube)

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Why do people spoof email?
+
+In many cases, the spoofed email is part of a **phishing** (scam) attack. In other cases, a spoofed email is used to dishonestly market an online service or sell you a bogus product. The intent is to trick the recipient into making a damaging statement or releasing sensitive information, such as passwords. If you're receiving bounced (returned) emails for messages that you never sent, this could be a case of spoofing.
+
+### Identify a spoofed message
+
+Scammers alter different sections of an email to disguise the *actual* sender of the message. To view the following properties, you must open the **email headers** of a message that you suspect has been spoofed. Following are some examples of spoofed properties:
+
+- **FROM** boss@companyexample.com - This appears to come from a legitimate source on a spoofed message.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: **FROM (for example, boss@companyexample.com)**

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+- **REPLY-TO** - This can also be spoofed, but a lazy scammer might leave the actual **REPLY-TO** address. If you see a different sending address here, the email might have been spoofed.
+
+- **RETURN-PATH** - This can also be spoofed, but a lazy scammer might leave the actual **RETURN-PATH** address. If you see a different sending address here, the email might have been spoofed.
+
+- **SOURCE IP** address or “X-ORIGIN” address - This is typically more difficult to alter but it is possible.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: **SOURCE IP address or “X-ORIGIN” address**

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+The first three properties can be easily altered by using settings in your Microsoft Outlook, Gmail, Hotmail, or other email software. The fourth property, IP address, can also be altered, but it usually requires more sophistication to make a false IP address convincing.
+
+In the following example, it appears that the recipient has received a message from their office assistant requesting money.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: In the following example, the recipient appears to have received

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+<img src="{% asset_path exchange/email-spoofing-best-practices-exchange-for-exchange-users/from_assistant.png %}" />
+
+The **subject** line should alert you immediately. This user should contact their assistant through another form of communication to confirm that they did not send this message. Next, you want to discover who actually sent the message by opening the message headers. It should look similar to the following:
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: The recipient should contact their assistant ... Next, discover who actually sent the message by opening the message headers. They should look similar to the following:

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+<img src="{% asset_path exchange/email-spoofing-best-practices-exchange-for-exchange-users/reply_to.png %}" />
+
+In this message header snippet, we see that the **From:** field shows the message being sent from **"Assistant"\<assistant@yourdomainexample.com\>**. However, we can also see that the **REPLY-TO:** field lists *spoofer@scam.com*. That is a clear example of a spoofed message.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: In this message header snippet, the **From:** field shows the message being sent from **"Assistant"\<assistant@yourdomainexample.com\>**. However, the **REPLY-TO:** field lists *spoofer@scam.com*. That is a clear example of a spoofed message.

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+You should blacklist any address that you find in the **REPLY-TO**, **RETURN-PATH**, and **SOURCE IP** field that is not an email address or IP address from which you normally receive mail.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: Blacklist any address that you find in the **REPLY-TO**, **RETURN-PATH**, and **SOURCE IP** field that is not an email address or IP address from which you normally receive mail.

(Maybe point readers to the Combat spoofing section for information on blacklisting.)

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+### Combat spoofing
+
+User education is the first line of defense against these types of attacks. If a user receives a spoofed message, they should perform the following tasks:
+
+ - Blacklist any email address or IP address listed in the **REPLY-TO**, **RETURN-PATH**, or **SOURCE IP** that you have determined to be fraudulent. For instructions, see [Block senders in Outlook Web App](/how-to/block-senders-in-owa).
+ - Immediately [change the password of your email account](/how-to/change-a-microsoft-exchange-mailbox-password) if you or your users provided that information at any point.
+ - Alert the rest of your business to the situation.
+
+Spoofing is possibly the most frustrating abuse issue to deal with, simply because it cannot be stopped. Spoofing is similar to hand-writing many letters, and signing someone else's name to it. You can imagine how difficult that would be to trace.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: handwriting (remove hyphen) many letters and signing (can remove comma)...

File: content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+The most impactful change you can make as an administrator is to implement **SPF**, **DKIM**, and **DMARC** records in that order. These are DNS records that add extra layers of protection to prevent malicious email from being sent out using your domain name.
```
Comment (cyrichardson) [inline content/exchange/email-spoofing-best-practices-exchange-for-exchange-users.md]: The most impactful change that you.....records, in that order.

...sent out with your domain name.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-eecb2c9f26/how-to/)

**Pull request merged**